### PR TITLE
VOTE: add sasha-tkachev to owners

### PR DESCRIPTION
**Pull request by sasha-tkachev**
Signed-off-by: Alexander Tkachev <sasha64sasha@gmail.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,3 +1,4 @@
 admins:
   - duglin
+  - sasha-tkachev
```
Comment (duglin) [inline OWNERS:3]: Can you please add @lionelvillard  (github: lionelvillard  ) too?

Comment (sasha-tkachev) [reply OWNERS]: done

**Comment by duglin** [comment]
LGTM
will open it up for all SDK maintainers to vote on this week's call.
Thanks @sasha-tkachev and @lionelvillard

**Review by lance — COMMENTED** [review]
/lgtm

**Comment by pierDipi** [comment]
LGTM

**Comment by jskeet** [comment]
LGTM

File: OWNERS [hunk 2]
```diff
@@ -1,3 +1,6 @@
 admins:
-  - duglin
+
```
Comment (duglin) [inline OWNERS:2]: can you remove the extra blank line?

**Comment by duglin** [comment]
Vote passed. Merging.
thanks @sasha-tkachev @lionelvillard

**Pull request merged**